packages/plugin-core/src/utils/LookupUtils.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  DendronError,
+  DEngineClient,
+  DVault,
+  NoteProps,
+  RespV3,
+  SchemaTemplate,
+  VaultUtils,
+} from "@dendronhq/common-all";
+import _ from "lodash";
+import { WSUtilsV2 } from "../WSUtilsV2";
+import { parseRef } from "./md";
+
+export class LookupUtils {
```

Comment:
can we place this file somewhere in `src/components/lookup` ? We have a bunch of other lookup utils there already; there's also already a file called `LookupUtils.test.ts` that's unrelated to this file.